ui/main.go

```diff
@@ -34,8 +34,38 @@ var (
 	yellow = conditionalColor(color.FgYellow)
 )
 
+// IsSilent returns true iff using silent logging level
+func IsSilent() bool {
+	return logLevel == logLevelSilent
+}
+
+// IsQuiet returns true iff output is limited (quiet ot silent logging level)
+func IsQuiet() bool {
+	return IsSilent() || logLevel == logLevelQuiet
+}
+
+// IsVerbose returns true iff usng verbose logging level
+func IsVerbose() bool {
+	return logLevel == logLevelVerbose
+}
+
+// SetSilent sets the logging kevel to silent
+func SetSilent() {
```

Comment:
Rather than having explicit getters and setters, if the log levels constants and `ui.Level` were exposed at the package level, we could drop the boilerplate here. 

The other packages (presumably `main`) would do something like:
```go
ui.Level = ui.LevelQuiet
```

Or, if there was a need for a setter, we could keep one total instead of one per level:
```go
ui.SetLevel(ui.LevelQuiet)
```